web/tests/functional/diff_local/test_diff_local.py

```diff
@@ -80,6 +80,25 @@ def test_new_json(self):
         for new_result in new_results:
             self.assertEqual(new_result['checkerId'], "core.NullDereference")
 
+    def test_non_existent_reports_directory(self):
+        """Hadles non existent directory well
+
+        Displays detailed information about base and new directories when
+        any of them are not exist.
+        """
+        error_output = ''
+        return_code = 0
+        try:
+            get_diff_results([self.base_reports], ['unexistent-dir-name'],
+                             '--new', 'json')
```

Comment:
No need to use the json output here because we do not want to postprocess the output:
```suggestion
                             '--new')
```

Reply:
Done.